src/datachain/sql/types.py

```diff
@@ -187,6 +187,22 @@ def on_read_convert(self, value, dialect):
         return read_converter(dialect).int32(value)
 
 
+class UInt32(Int):
```

Comment:
[Q] Does this need to be added? (now that it isn't being used - seems unrelated to this PR)

Reply:
It doesn't, but now that I've already added it, I feel bad removing it :D ... I think we actually had the issue to add it anyway (cannot find it atm) as it's kind of missing (we have support for `int32`, `int64` and `uint64`).

Reply:
Can you see how this is unrelated to this PR?

Reply:
Of course I can see, If you really want this to be by PS I will split into 2 PRs, not a problem, but I wanted to avoid that additional work in this relatively small issue...that was my point.

Reply:
Split into separate PR https://github.com/iterative/datachain/pull/568